.github/workflows/deploy.yml

```diff
@@ -11,9 +11,11 @@ jobs:
       if: github.event.action == 'deploy'
       env:
         FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+        GTM_CONTAINER_ID: ${{ secrets.GTM_CONTAINER_ID }}
       run: |
         npm install firebase-tools
         PATH=$(npm bin):$PATH
         npm install
         cd functions && npm install && cd ..
-        firebase deploy --token "$FIREBASE_TOKEN"
+        firebase deploy
```

Comment:
`--token` does not need if `FIREBASE_TOKEN` env is set.
Refer: https://firebase.google.com/docs/cli#cli-ci-systems
> Store the token as the environment variable FIREBASE_TOKEN. Your system will automatically use the token.